bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -479,4 +499,27 @@ async function deleteRecipient({ client, dataStore, integrationParameters, actio
         throw new Error(await response.text())
     }
     dataStore.deleteById('envelopes_recipients_signers', actionParameters.recipient_Id)
+}
+async function envelopeview({ client, dataStore, integrationParameters, actionParameters }) {
+    const ACCOUNT_ID = integrationParameters.account_Id;
+    const envelopeViewResponse = await client.fetch(`accounts/${ACCOUNT_ID}/envelopes/${actionParameters.envelope_Id}/views/recipient`,
+        {
+            method: "POST",
+            body: JSON.stringify(
+                {
+                    "userName": `${actionParameters.userName}`,
+                    "email": `${actionParameters.email}`,
```

Comment:
userName and email are already strings, so, why to put it to ``?
```suggestion
                    "userName": actionParameters.userName,
                    "email": actionParameters.email,
```